tools/clang/test/DXC/embed_dbg.test

```diff
@@ -1,7 +1,7 @@
 // Embed debug info
 // RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zi /Qembed_debug /Fo %t.embedpdb
 // RUN: %dxc -dumpbin %t.embedpdb | FileCheck  %s --check-prefix=EMBED_PDB
-// RUN: not file -E *.pdb
+// RUN: not ls *.pdb && count 0
```

Comment:
This should be `not ls *.pdb | count 0` rather than `&&` - count reads its standard input

Reply:
Fixed.